gradle.properties

```diff
@@ -0,0 +1,26 @@
+# Available idea versions:
+# https://www.jetbrains.com/intellij-repository/releases
+# https://www.jetbrains.com/intellij-repository/snapshots
+
+
+ideaVersion = 213.6461.79
+intellijGradlePluginVersion = 1.3.0
+jupiterEngineVersion = 5.8.2
+jupiterApiVersion = 5.8.2
+changelogVersion = 1.3.1
+javaVersion = 11
+
+
+
+
+# rfc-parser properties
+# https://mvnrepository.com/artifact/com.google.guava/guava
```

Comment:
what is purpose of this comment?

Reply:
Because of transparency. If the future shows that rfc-parser will not be usable, then we will simply know which versions we will delete and the second one is for easy finding the latest version of guava.

Reply:
I definitely agree with # rfc-parser properties but link to guava :) that is easily findable searching "maven guava" so i really dont think that it is necessary to put it there